content/docs/gke/deploy/deploy-ui.md

```diff
@@ -9,23 +9,85 @@ deploy Kubeflow on GCP. If you prefer to use the
 command line, see the guide to 
 [deployment using the CLI](/docs/gke/deploy/deploy-cli).
 
-1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
-  in your web browser.
-1. Sign in using a GCP account that has administrator privileges for your 
-  GCP project.
-1. Complete the form, following the instructions on the left side of the form.
-  In particular, ensure that you enter the same **deployment name** as you used
-  when creating the OAuth client ID.
-1. Check **Skip IAP** box if you want to use basic auth.
-1. Click **Create Deployment**.
+## Overview of the deployment user interface (UI)
 
-Here's a partial screenshot of the deployment UI, showing all the fields in the 
-form:
+Here's a partial screenshot of the deployment UI:
 
 <img src="/docs/images/kubeflow-deployment.png" 
   alt="Kubeflow deployment UI"
   class="mt-3 mb-3 border border-info rounded">
 
+Follow these steps to open the deployment UI and deploy Kubeflow on GCP:
+
+1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
+  in your web browser. You should see a form like the one in the above
+  screenshot.
+1. Sign in to your browser using a GCP account that has administrator privileges 
```

Comment:
How about
that has `project owner role` for your GCP project
?

Reply:
Done.